### PR TITLE
No smooth

### LINKED ISSUE #2321: ENH: Disable gaussian smoothing in mne.viz.plot_topo_image_epochs
Hello,

I'd like to disable the gaussian smoothing when plotting image epochs, so I can clearly identify which trials have artifacts in them. I thought that the user could set ```sigma=None```, in which case the smoothing step is skipped. Would anyone else be interested in this option?

Alan

### PR DESCRIPTION
**Pull request by leggitta**
Sets ```interpolation='nearest'``` for plotting image epochs (since these use gaussian filtering). 

resolve #2321

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Can you pop in a couple before/after shots for the two functions you changed?

**Comment by leggitta** [comment]
Sure!

from mne.viz.plot_topo_image_epochs, before (interpolation undefined, sigma=0)
screenshot from 2015-07-21 10 38 57

from mne.viz.plot_topo_image_epochs, after (interpolation='nearest', sigma=0)
screenshot from 2015-07-21 10 38 09

from mne.viz.plot_image_epochs, before (interpolation undefined, sigma=0)
screenshot from 2015-07-21 10 44 39

from mne.viz.plot_image_epochs, after (interpolation='nearest', sigma=0)
screenshot from 2015-07-21 10 45 38

**Comment by Eric89GXL** [comment]
LGTM. I think we should change the default to `sigma=0.` or to something that is actually visible like `sigma=1.`. `sigma=0.3` just doesn't seem to do much, right?

**Comment by leggitta** [comment]
Yeah, this is with sigma=0.3 and its almost identical to sigma=0
screenshot from 2015-07-21 10 57 53

I was also wondering if I could code it so that if you set sigma=None, it skips the gaussian filter altogether.

**Comment by Eric89GXL** [comment]
+1 for changing the default to `sigma=0.` and skipping the filter in that case

**Comment by leggitta** [comment]
woops, that was an accidental close

**Comment by jona-sassenhagen** [comment]
*Somebody* should do something about the incorrect x axis for the evoked trace.

File: mne/viz/epochs.py [hunk 1]
```diff
@@ -39,9 +39,9 @@ def plot_image_epochs(epochs, picks=None, sigma=0.3, vmin=None,
     picks : int | array-like of int | None
         The indices of the channels to consider. If None, all good
         data channels are plotted.
-    sigma : float
+    sigma : float | None
         The standard deviation of the Gaussian smoothing to apply along
-        the epoch axis to apply in the image.
+        the epoch axis to apply in the image. If None, no smoothing is applied.
```
Comment (agramfort) [inline mne/viz/epochs.py]: what is the difference between None and 0?

Comment (Eric89GXL) [inline mne/viz/epochs.py]: agreed, I think `sigma=0.` should mean don't apply any filtering

**Comment by leggitta** [comment]
Done. I think I could also address @jona-sassenhagen 's concern by fixing the x-limit for that 2nd axis. Should I do that in this PR?

**Comment by Eric89GXL** [comment]
Sure, I think +11/-8 is small enough to add a few more lines. Kind of makes me miss your old GSoC PR monstrosities :)

**Comment by leggitta** [comment]
I know, it seems almost too simple. I feel like I should at least throw in a few trailing white spaces for someone to find.

Now the x-limit on the evoked plot matches the epoch image.

screenshot from 2015-07-21 16 15 51

**Comment by Eric89GXL** [comment]
Thanks @leggitta +1 for merge from me

**Comment by agramfort** [comment]
thanks @leggitta !

**Pull request merged**